user_profile.py

```diff
@@ -56,12 +56,181 @@
+    'last_name' : fields.String,
+    'age' : fields.Integer,
+    'date_of_birth' : fields.DateTime(dt_format='iso8601'),
+    'address_line_1' : fields.String,
+    'address_line_2' : fields.String,
+    'city' : fields.String,
+    'province' : fields.String,
+    'country' : fields.String,
+    'postal_code' : fields.String,
+    'gender' : fields.String,
+    'religion' : fields.String,
+    'profile_image' : fields.String,
+    'user_bio' : fields.String,
+    'interests' : fields.String,
+    'education_institutions' : fields.String,
+    'education_majors' : fields.String,
+    'education_degrees' : fields.String,
+    'graduation_date' : fields.DateTime(dt_format='iso8601'),
+    'identification_option' : fields.String,
+    'identification_material' : fields.String,
+    'user_id' : fields.String
+}
+
+
+
+
 
 # Querying and inserting into user profile database Flask_Restful API
 class UserInformationResource(Resource):
     def __init__(self) -> None:
         super().__init__()
 
+    # private method to add argument into the form data
+    def __formData_addArguments(self, formData) -> None:
+        formData.add_argument(
+            "firstName",
+            type=str,
+            help="First name is required",
+            required=True,
+        )
+        formData.add_argument("midName", type=str, required=False)
+        formData.add_argument(
+            "lastName",
+            type=str,
+            help="Last name is required",
+            required=True,
+        )
+        formData.add_argument(
+            "age",
+            type=int,
+            help="Age is required and must be larger than 18",
+            required=True,
+        )
+        formData.add_argument(
+            "birthDay",
+            type=inputs.date,
+            help="Birthday is required and the format must be YYYY-MM-DD",
+            required=True,
+        )
+        formData.add_argument(
+            "firstAddress",
+            type=str,
+            help="First address is required",
+            required=True,
+        )
+        formData.add_argument(
+            "secondAddress",
+            type=str,
+            help="Second address is required",
+            required=False,
+        )
+        formData.add_argument(
+            "city", type=str, help="City is required", required=True
+        )
+        formData.add_argument(
+            "province", type=str, help="Province is required", required=True
+        )
+        formData.add_argument(
+            "country", type=str, help="Country is required", required=True
+        )
+        formData.add_argument(
+            "postalCode",
+            type=str,
+            help="Postal Code is required and maximum of 10 characters long",
+            required=True,
+        )
+        formData.add_argument(
+            "gender", type=str, help="Gender is required", required=True
+        )
+        formData.add_argument(
+            "profile_image",
```

Comment:
So this field is snake case, but `postalCode`, `firstAddress`, and other fields are camel case.

Let's use snake case here to follow Python convention. The client can change the field name easily to send the correct data to the server.

`formData` is also in camelcase, let's change that to snake case: `form_data`